Makefile

```diff
@@ -197,9 +197,15 @@ $(TIMESTAMPS_DIR)/manifests: $(GO_SOURCES)
 manifests: CRD_OPTIONS ?= "crd:crdVersions=v1,ignoreUnexportedFields=true"
 manifests: fmt controller-gen $(TIMESTAMPS_DIR)/manifests ## Generate manifests e.g. CRD, RBAC etc.
 
+$(TIMESTAMPS_DIR)/golangci-lint:
```

Comment:
Do we want to rely on latest for golangci-lint?
From my experience, golangci-lint sometimes tweaks the way it handles the config leading to failures and requiring updates in the config

Reply:
Good point, I can probably lock the version in the makefile, so we chose when we upgrade.
On the other hand, we will have to remember to upgrade eventually!

Reply:
done